modeshape-persistence-tests/pom.xml

```diff
@@ -0,0 +1,442 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.modeshape</groupId>
+        <artifactId>modeshape-parent</artifactId>
+        <version>5.2-SNAPSHOT</version>
+        <relativePath>../modeshape-parent</relativePath>
+    </parent>
+
+    <!-- The groupId and version values are inherited from parent -->
+    <artifactId>modeshape-persistence-tests</artifactId>
+    <packaging>jar</packaging>
+    <name>ModeShape Persistence Tests</name>
+    <description>ModeShape persistence tests</description>
+    <url>http://www.modeshape.org</url>
+    
+    <properties>
+        <database.name>modeshape_db</database.name>
```

Comment:
these seem to only be used for MySQL and Postgres and not for Oracle. Also, it seems a bit suspect that we're using `system` and `oracle` as values.